File: 0001 Code Chef/0021 Masterchief final/implementation/0010 nodejs/master_chef.js
const readline = require('readline');

const rl = readline.createInterface({
    input:process.stdin,
    output:process.stdout
});


rl.question('',(input)=>{
    let test = parseInt(input);
    let count = 0;

    rl.on('line',(input)=>{
        let rank = parseInt(input);
        if(rank >= 1 && rank <= 10){
            console.log("YES");
        }else{
            console.log("NO");
        }
        count = count + 1;
        if(count == test){
            rl.close();
        }
    })
})

File: 0001 Code Chef/0020 Masterchief final/implementation/0002 c++/master_chef.cpp
#include <iostream>

int main(){
    int test = 0;
    std::cin >> test;
    while(test > 0){
        int rank;
        std::cin >> rank;
        if(rank <= 10){
            std::cout << "YES" << std::endl;
        }else{
            std::cout << "NO" << std::endl;
        }
        test = test - 1;
    }
    return 0;
}